### ISSUE TITLE
Is there going to be another update any time soon?

### ISSUE THREAD

**Issue by Husain136** [Unity-Technologies/FPSSample#83]

**Comment by n8allan** [comment]
Yes, Unity 2018.3 is getting a little long in the tooth, and I've tested the viability of moving this project to newer versions and there are significant obstacles with the dependencies.  More frequent drops would be a huge help to the community.

**Comment by twobob** [comment]
no. A new project will come out soon. This one is mothballed.

**Comment by bcfowler1** [comment]
is there an equivalent of this as of Mar 2020?

**Comment by PixelGM** [comment]
No, Unity has 'Finished' this project.

**Comment by Stexe** [comment]
> 
> 
> No, Unity has 'Finished' this project.

[citation needed]
